src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  max-width: $day-size*7 + $days-gap*6;
+  gap: $days-gap;
+  background-color: #fff;
+
+  @each $day, $dayNum in
+    ("mon": 0, "tue": 1, "wed": 2, "thu": 3, "fri": 4, "sat": 5, "sun": 6) {
```

Comment:
move to `("mon": 0, "tue": 1, "wed": 2, "thu": 3, "fri": 4, "sat": 5, "sun": 6)` a variable

Reply:
resolved